config.md

```diff
@@ -59,6 +59,7 @@ For Solaris, the mount entry corresponds to the 'fs' resource in the [zonecfg(1M
 
 * **`destination`** (string, REQUIRED) Destination of mount point: path inside container.
   This value MUST be an absolute path.
+  * Posix: the path and child directories MUST exist, a runtime MUST NOT create directories automatically to a mount point.
```

Comment:
To be clear then, you recommend removing the addition both here, and keep the removal in config.go.